caffe2/python/onnx/backend.py

```diff
@@ -506,9 +506,16 @@ def make_rnn(direction_offset):
                 pred_mh.net.Concat([outputs_f[i], outputs_b[i]],
                                    [n.outputs[i], cls.dummy_name()], axis=0)
 
-        return Caffe2Ops(list(pred_mh.Proto().op),
-                         list(init_net.Proto().op),
-                         list(pred_mh.Proto().external_input))
+        initializers = {i.name for i in init_model.graph.initializer}
+        outputs = {output for node in init_model.graph.node for output in node.output}
+        has_initializers = all(x in initializers or x in outputs for x in (W, R, B))
```

Comment:
Could you add a comment here saying this `has_initializers` should be always true in reality?